addons/account/models/account_tax.py

```diff
@@ -343,6 +349,7 @@ def _sanitize_vals(self, vals):
                     else (command, id, v)
                     for command, id, v in sanitized.pop(fname)
```

Comment:
Not related to the PR but I'm wondering why we make a distinction.
Also, I think in case of `Command.LINK`, it's possible to only have `(Command, id)` and no `v`.

edit: But I guess it doesn't happens because those comment are made automatically when parsing the CSV

Reply:
> Not related to the PR but I'm wondering why we make a distinction.

Because the third element of the tuple is not a dict in the other cases

> But I guess it doesn't happens because those comment are made automatically when parsing the CSV

Indeed